### PR TITLE
fixed the local compilation issues for newly created module

### PR DESCRIPTION
**Pull request by jagankumar-egov**

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js [hunk 1]
```diff
@@ -4,23 +4,11 @@ import { useTranslation } from "react-i18next";
 import { useRouteMatch } from "react-router-dom";
 import CitizenApp from "./pages/citizen";
 import Create from "./pages/citizen/create/index";
-import EmployeeApp from "./pages/employee";
-import BrSelectName from "./pagecomponents/BrSelectName";
-import BRSelectPhoneNumber from "./pagecomponents/BrSelectPhoneNumber";
-import BRSelectGender from "./pagecomponents/BRSelectGender";
-import BRSelectEmailId from "./pagecomponents/SelectEmailId";
-import BRSelectPincode from "./pagecomponents/BRSelectPincode";
-import BrSelectAddress from "./pagecomponents/BrSelectAddress";
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js:13]: these files doesn't exists but import was added.
First create a module with one component and enable it.

Comment (jagankumar-egov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js:13]: which causes compilation issues

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js [hunk 2]
```diff
@@ -34,7 +22,7 @@ export const FSModule = ({ stateCode, userType, tenants }) => {
         return <CitizenApp path={path} stateCode={stateCode} />;
     }
 
-    return <EmployeeApp path={path} stateCode={stateCode} />;
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js:37]: these import doesn't exist

File: frontend/micro-ui/web/micro-ui-internals/package.json [hunk 3]
```diff
@@ -28,11 +29,9 @@
     "dev:example": "cd example && yarn start",
     "dev:sample": "cd packages/modules/sample && yarn start",
     "dev:mukta": "cd packages/Mukta && yarn start",
-    "dev:pt": "cd packages/modules/pt && yarn start",
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/package.json:31]: removed pt module

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js [hunk 4]
```diff
@@ -4,23 +4,11 @@ import { useTranslation } from "react-i18next";
 import { useRouteMatch } from "react-router-dom";
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js:4]: Your package.json says the source file is Module.js
https://github.com/Osamai8/DIGIT-OSS/blob/91076f3e80166605324f25c02ee4b829ccbcf4e8/frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/package.json#L8

But the file name was modules.js which made it not to get compiled

**Review by jagankumar-egov — COMMENTED** [review]
Resolved local compilation issues

**Comment by jagankumar-egov** [comment]
Screenshot 2023-07-26 at 12 32 32 PM
Screenshot 2023-07-26 at 12 31 15 PM

**Comment by Osamai8** [comment]
Thanks Jagan for your help in solving this problem.

**Pull request merged**